packages/testcontainers/src/containers/DeFiDContainer.ts

```diff
@@ -35,7 +35,7 @@ export abstract class DeFiDContainer extends DockerContainer {
     if (process?.env?.DEFICHAIN_DOCKER_IMAGE !== undefined) {
       return process.env.DEFICHAIN_DOCKER_IMAGE
     }
-    return 'defi/defichain:2.6.2'
+    return 'defi/defichain:HEAD-36ae648'
```

Comment:
Since this is going to change as time goes on, is there a better default image we can use?

Reply:
Finally this will be changed to an official image, but in this case, backend implementation has been done in a separate branch in Ain side. therefore we have to a specific image.